### ISSUE TITLE
Website changelog does not reflect CHANGELOG.md

### ISSUE THREAD

**Issue by joshy** [voila-dashboards/voila#1503]
### Problem
The latest changelog entry on the website https://voila.readthedocs.io/en/stable/changelog.html is 0.5.1 but now 0.5.8 is already released: https://github.com/voila-dashboards/voila/blob/main/CHANGELOG.md

### Suggested Improvement
Maybe it is possible to update the website with each release or the website could always render CHANGELOG.md?

**Comment by martinRenou** [comment]
Thanks for opening an issue, looking into it!

**Comment by jtpio** [comment]
It's probably because there is a `stable` branch on the repo? https://github.com/voila-dashboards/voila/tree/stable

**Comment by martinRenou** [comment]
Yes, I updated the stable branch, it built and passed! Thanks https://voila.readthedocs.io/en/stable/

**Comment by jtpio** [comment]
Normally the `stable` branch is reset after each release.

Maybe it's related to the switch

**Comment by jtpio** [comment]
Reopening so we should update that hook to the branch keeps being updated.

**Comment by jtpio** [comment]
An alternative would be to remove that `stable` branch, since RTD should manage stable versions automatically?

**Comment by martinRenou** [comment]
> should manage stable versions automatically?

Does it? I understand that by default "latest" is the latest commit?

**Comment by jtpio** [comment]
From https://docs.readthedocs.io/en/stable/versions.html:

> If your project has any tags or branches with a name following [semantic versioning](https://semver.org/), we also create a stable version tracking your most recent release. If you want a custom stable version, create either a tag or branch in your project with that name.
